src/pages/Governance/utils.tsx

```diff
@@ -122,3 +123,25 @@ export function renderStatusIcon(
       return assertNever(proposalState);
   }
 }
+
+export function getExecutionStatusColor(isResolved: boolean): string {
+  return isResolved ? primaryColor : warningColor;
+}
+
+export function renderExecutionStatusIcon(isResolved: boolean): JSX.Element {
```

Comment:
For stuff like this, just make it a component. `export function ExecutionStatusIcon({isResolved}: {isResolved: boolean}) { ... }`